src/components/ui/ControlBar/ControlBarItem.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import React, { FC, ReactNode } from 'react';
+import React, { FC, ReactNode, useMemo } from 'react';
```

Comment:
Should we update tooltip section in the storybook for all these components.

Reply:
The related tooltip props should be showing up in the props table of each component. Do you want an additional section detailing just the tooltip piece?

Reply:
No i meant the existing docs: https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/docs/ui-components-controlbar--control-bar
```
ControlBars support tooltips on any buttons by adding a data-tooltip property. Adding a data-tooltip-position property controls the tooltip position
```
We should remove this and redirect to the new story you wrote below or update it to include passing the containerId nd tooltipText

Reply:
ah didn't see those, I'll update